tools/acrn-crashlog/usercrash/README.rst

```diff
@@ -53,19 +53,11 @@ The work flow diagram:
 Usage
 *****
 
-- The server is launched automatically at boot, and the client is configured in
-  ``core_pattern`` or ``coredump-wrapper``. In ``prepare.service``, it will
-  check the content of ``/proc/sys/kernel/core_pattern``. If there is
-  ``coredump-wrapper``, which means that ``core_pattern`` has been set in
-  ``systemd``, no need to do it again. Otherwise, the content should be
-  changed by:
-
-.. code-block:: none
-
-   $ echo "|/usr/bin/usercrash_c %p %e %s" > /proc/sys/kernel/core_pattern
-
-That means client will be triggered once userspace crash occurs. Then the
-event will be sent to server from client.
+- The server is launched automatically at boot after this tool is enabled, and
```

Comment:
When you say "this tool is enabled", I assume you mean once the user has run `sudo crashlogctl enable`? If correct, you need to explicitly say that here because it's in a different document than the one above and there is no real context anymore as to what"this tool is enabled" really means.

Reply:
Yes, you're right.
I'll correct it like:
this tool is enabled with instruction ``sudo crashlogctl enable``, and ...